internal/host/validator.go

```diff
@@ -614,3 +626,25 @@ func isInvalidImageStatus(imageStatus *models.ContainerImageAvailability) bool {
 	return imageStatus.Result == models.ContainerImageAvailabilityResultFailure ||
 		(imageStatus.SizeBytes > 0 && imageStatus.DownloadRate < ImageStatusDownloadRateThreshold)
 }
+
+func (v *validator) sufficientDiskSpeed(c *validationContext) ValidationStatus {
```

Comment:
I would change that to `currentBootDeviceHasntFailed`
In addition, please document that this validation is relevant for multiple-installation-attempts when we don't want to allow the user to use the same disk that has failed in the previous installation attempt.